### PR TITLE
File find popup up/down keys resolves #977

### LINKED ISSUE #977: Fuzzy Finder: Move up/down with vim keys
**Is your feature request related to a problem? Please describe.**
- I also don't know how to scroll to other selections -- `j`/`k` don't work given the context.

**Describe the solution you'd like**
- I'd like to be able to select other matches. 

**Describe alternatives you've considered**
If it's already possible to have multiple key mappings per command, then I can simply do that and map `C-j` and `C-k` for this. If not, I guess the solution would be to split out `move_up` and `move_down` to read-only lists and "picker" style lists.

### PR DESCRIPTION
**Pull request by Curiosidad-Racional**
This Pull Request fixes/closes #977.

It changes the following:
- adds up and down key for file find popup

I followed the checklist:
- [ ] I added unittests
- [x] I ran `make check` without errors
- [X] I tested the overall application
- [x] I added an appropriate item to the changelog

### CODE REVIEW

File: vim_style_key_config.ron [hunk 1]
```diff
@@ -44,6 +44,8 @@
     end: ( code: End, modifiers: ( bits: 0,),),
     move_up: ( code: Char('k'), modifiers: ( bits: 0,),),
     move_down: ( code: Char('j'), modifiers: ( bits: 0,),),
+    popup_up: ( code: Char('p'), modifiers: ( bits: 2,),),
```
Comment (extrawurst) [inline vim_style_key_config.ron:47]: shouldn't this use arrow keys because p/n will be used as text input in the fuzzy finder?

Comment (Curiosidad-Racional) [reply vim_style_key_config.ron:47]: As you know, `( bits: 2,)` means control key, then `C-p/C-n` will be used for up and down, and `p/n` would be typed.

Comment (extrawurst) [reply vim_style_key_config.ron:47]: The problem with these modifiers is that they are notoriously inconsistently supported in the various different terminals

Comment (extrawurst) [reply vim_style_key_config.ron:47]: And additionally they often get picked up as special commands - but hey I am no expert with the vim key bindings - I trust you there 👍

**Comment by extrawurst** [comment]
Please add an appropriate entry to the changelog

File: vim_style_key_config.ron (hunk 1 above)
Comment (Curiosidad-Racional) [reply vim_style_key_config.ron:47]: Thank you

**Comment by Curiosidad-Racional** [comment]
> Please add an appropriate entry to the changelog

Added entry and resolved conflicts.

**Comment by extrawurst** [comment]
Thank you @Curiosidad-Racional 👍

**Pull request merged**